linter/src/rules/adding_field_with_default.rs

```diff
@@ -106,6 +106,7 @@ ALTER TABLE "core_recipe" ADD COLUMN "foo" integer DEFAULT uuid();
         let ok_sql = r#"
 -- NON-VOLATILE
 ALTER TABLE "core_recipe" ADD COLUMN "foo" integer DEFAULT 10;
+ALTER TABLE "account" ADD COLUMN "last_modified" timestamptz DEFAULT now();
```

Comment:
our code for checking volatility of functions doesn't work. now() is stable, while a function like random is volatile.

```sql
select
  proname,
  pronamespace::regnamespace,
  provolatile
from pg_proc
where proname = 'now'
  or proname = 'random';
```

I think we could hard code in a list of functions and then warn when we can't know for certain with a more detailed error message.